### PR TITLE
Add small section on Spicy.

### PR DESCRIPTION
**Pull request by bbannier**

### CODE REVIEW

File: devel/spicy-analyzers.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+and `spicy-plugin <https://github.com/zeek/spicy-plugin>`_,
+respectively.
+
+Writing an analyzer
+===================
+
+Analyzer scaffolding including a Spicy grammar ``.spicy``, Zeek integration
+glue code ``.evt`` and a CMake build setup can be generated with the `zkg
+package manager <https://docs.zeek.org/projects/package-manager>`_ with the
+default package template by passing ``--feature spicy-analyzer``.
```
Comment (bbannier) [inline devel/spicy-analyzers.rst:30]: This still requires zeek/package-template#1.

File: devel/spicy-analyzers.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+with `spicy-plugin <https://github.com/zeek/spicy-plugin>`_ so that one can
+create Zeek protocol, packet and file analyzers.
+
+Spicy is :doc:`documented separately <spicy:index>`, so this section just
+presents high-level points relevant for Zeek.
+
+Installation
+============
+
+A Zeek configured with default ``./configure`` options bundles Spicy. If the
```
Comment (rsmmr) [inline devel/spicy-analyzers.rst:17]:
```suggestion
A Zeek configured with default ``./configure`` options includes Spicy. If the
```

**Review by rsmmr — APPROVED** [review]

**Comment by bbannier** [comment]
Let's wait with merging this until the actual zkg package template support zeek/package-template#1, and the Spicy feature itself zeek/zeek#2006 are merged.

**Comment by ckreibich** [comment]
I merged this with a small tweak to avoid

    /home/christian/devel/zeek/zeek-docs/devel/spicy-analyzers.rst:11: WARNING: unknown document: spicy:index

I just made that construct plaintext since the section already links to the Spicy docs in other spots.

**Pull request merged**